src/dependency-manager/src/graph/node/index.ts

```diff
@@ -19,6 +19,10 @@ export abstract class DependencyNode implements DependencyNodeOptions {
       .replace(/\//g, '.');
   }
 
+  get namespace_ref() {
```

Comment:
Once we have everything working lets readdress normalized_ref to work across docker, k8s, ecs, and parameter prefix.